src/apphosting/githubConnections.ts

```diff
@@ -117,21 +136,30 @@ export async function linkGitHubRepository(
     installationId = await promptGitHubInstallation(projectId, location, oauthConn);
   }
 
-  let connectionMatchingInstallation = await getConnectionForInstallation(
```

Comment:
getting rid of this would mean we'll be making duplicate connections for the same github account/org when we can re-use them.

Reply:
Yeah, good point. Didn't mean to remove that